paimon-web-common/src/main/java/org/apache/paimon/web/common/result/FetchResultParams.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.common.result;
+
+/** Represents the parameters required to fetch the results of a certain operation. */
+public class FetchResultParams {
+
+    private final String sessionId;
+    private final String submitId;
+    private final Long token;
+
+    private FetchResultParams(Builder builder) {
```

Comment:
I don't think the constructor need to use builder as parameter. The builder mode isn't used via this way.

Reply:
Yes, I will modify this class.